.drone.star

```diff
@@ -1,46 +1,88 @@
+
+TAG_PATTERN = "${DRONE_TAG:-${DRONE_COMMIT_SHA:0:7}}"
 def main(ctx):
-  return {
+  pl = {
     "kind": "pipeline",
     "name": "default",
-    "steps": [
-      {
-        "name": "build web ui",
-        "image": "node:12.13.0-stretch",
-        "commands": ["make web/dist"]
+    "steps": []
+  }
+
+  pl["steps"].append({
+    "name": "build web ui",
+    "image": "node:12.13.0-stretch",
+    "commands": ["make web/dist"]
+  })
+  pl["steps"].extend(build(ctx, "linux", "amd64"))
+  pl["steps"].extend(build(ctx, "linux", "arm64"))
+  pl["steps"].extend([
+    {
+      "name": "make release",
+      "image": "alpine",
+      "commands": [
+        "apk add upx",
+        "upx -9 bin/*",
+        "mkdir -p release",
+        "cp bin/* release",
+        "cp script/install.sh release",
+        "cp script/uninstall.sh release"
+      ]
+    },
+    {
+      "name": "github release",
+      "image": "plugins/github-release",
+      "settings": {
+        "api_key": {"from_secret": "github_api_key"},
+        "files": "release/*",
+	    "checksum": ["sha256"]
```

Comment:
This line's indentation contains mixed spaces and tabs.

Reply:
Here seems to have been like this before. Could you fix this?

Reply:
fixed